### PR TITLE
Create "at x measures" block

### PR DESCRIPTION
**Pull request by bencodeorg**
Same as existing "after x seconds/measures" block, except that, when measures are selected, the behavior change is observed at the beginning of the selected measure rather than the end.

Open to ideas on what to name this block :).

Companion PR in the dance-party repo here: https://github.com/code-dot-org/dance-party/pull/660

https://github.com/code-dot-org/code-dot-org/assets/25372625/d408f0ba-9500-45e7-a79a-d87d7ac98a57

## Links

- trello: https://trello.com/c/BNkTW68q/71-nov-2-add-an-at-measure-event-block-to-dance

## Testing story

Tested manually that behavior changes were observed when a given measure started, rather than when it ended. Also checked that blocks still worked when seconds selected rather than measures.

### CODE REVIEW

File: dashboard/app/views/home/_levelbuilder.html.haml [hunk 1]
```diff
@@ -5,7 +5,7 @@
       = link_to 'Scripts', scripts_path
       = link_to '(Update all)', scripts_path(rake: '1'), data: { confirm: 'Are you sure want to re-seed script data?'}
     %li= link_to t('builder.manage'), levels_path
-    %li= link_to 'Pools', pools_path
+    %li= link_to 'Block Pools', pools_path
```
Comment (bencodeorg) [inline dashboard/app/views/home/_levelbuilder.html.haml:8]: Unrelated, but I remember looking at this admin menu for years before working on Blockly and wondering what are "pools"??

File: dashboard/config/blocks/Dancelab/Dancelab_atTimestampNotAfter.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "category": "Events",
+  "config": {
+    "style": "event_blocks",
+    "func": "atTimestampNotAfter",
```
Comment (fisher-alice) [inline dashboard/config/blocks/Dancelab/Dancelab_atTimestampNotAfter.json:5]: Is there already an `atTimestamp` function defined?

Comment (fisher-alice) [reply dashboard/config/blocks/Dancelab/Dancelab_atTimestampNotAfter.json:5]: Just took a look at your other `dance-party` PR and see there is.

**Review by sanchitmalhotra126 — APPROVED** [review]

**Review by fisher-alice — APPROVED** [review]
LGTM!

**Pull request merged**